pydantic_ai_slim/pydantic_ai/models/openai.py

```diff
@@ -637,20 +637,26 @@ def _process_thinking(self, message: chat.ChatCompletionMessage) -> list[Thinkin
 
         This method may be overridden by subclasses of `OpenAIChatModel` to apply custom mappings.
         """
+        profile = OpenAIModelProfile.from_profile(self.profile)
+        custom_field = profile.openai_chat_custom_reasoning_field or ''
```

Comment:
Let's not have this default to an empty string, that's just going to lead to confusion. I'd rather have it be `None`, and then deal with that in the `for...` below